crates/rspack/src/lib.rs

```diff
@@ -1,5 +1,64 @@
 #![forbid(missing_docs)]
 
-//! The Rspack compiler.
-
+//! Rspack is a high performance JavaScript bundler written in Rust. It offers strong compatibility with the webpack ecosystem, allowing for seamless replacement of webpack, and provides lightning fast build speeds.
+//!
+//! ## Why rspack?
```

Comment:
Can we link these introduction to https://rspack.dev/ to reduce maintenance overhead.